src/components/LoginForm/index.js

```diff
@@ -68,14 +35,67 @@ function Login({ providers, ...props }) {
   }, [session]);
 
   return (
-    <Grid
-      container
-      justify="space-around"
-      alignItems="center"
-      className={classes.root}
-    >
-      <Grid item xs={12} className={classes.item}>
-        <form noValidate className={classes.formStyles}>
+    <Section className={classes.section}>
+      <Grid container>
+        <Grid item xs={7} className={classes.container}>
+          <Typography variant="h2">{title}</Typography>
+          <Typography className={classes.text}>{description}</Typography>
+          <form className={classes.form}>
+            <Grid container>
+              <Grid item xs={12}>
+                <TextField
+                  className={classes.textfield}
+                  InputLabelProps={{ className: classes.label }}
+                  InputProps={{ className: classes.input }}
+                  autoComplete="email"
+                  name="email"
+                  variant="outlined"
+                  fullWidth
+                  id="email"
+                  label="Email"
+                  autoFocus
+                />
+              </Grid>
+              <Grid item xs={12}>
+                <TextField
+                  className={classes.textfield}
+                  InputLabelProps={{ className: classes.label }}
+                  InputProps={{
+                    className: classes.input,
+                    endAdornment: (
+                      <InputAdornment position="end">
+                        <Image height={45} width={45} src={ViewIcon} alt="" />
+                      </InputAdornment>
+                    ),
+                  }}
+                  variant="outlined"
+                  margin="normal"
+                  fullWidth
+                  name="password"
+                  label="Password"
+                  type="password"
+                  id="password"
+                  autoComplete="current-password"
+                />
+              </Grid>
+            </Grid>
+          </form>
+          <div>
+            <Button
+              className={classes.button}
+              variant="contained"
+              color="primary"
+            >
+              Edit
+            </Button>
+            <Button
+              className={classes.button}
+              variant="contained"
+              color="primary"
+            >
+              Delete
+            </Button>
```

Comment:
While this is what is shown in the Figma design, I believe it's wrong. What are you supposed to edit or delete here. I think there should just be one **Log in** or **Sign in** button.